nav2_costmap_2d/plugins/obstacle_layer.cpp

```diff
@@ -571,8 +571,11 @@ ObstacleLayer::raytraceFreespace(
   if (!worldToMap(ox, oy, x0, y0)) {
     RCLCPP_WARN(
       logger_,
-      "Sensor origin at (%.2f, %.2f) is out of map bounds. The costmap cannot raytrace for it.",
-      ox, oy);
+      "Sensor origin at (%.2f, %.2f) is out of map bounds (%.2f x %.2f at %.2f, %.2f). "
```

Comment:
Can this be more explicit that the second set of numbers are the origin? I'm not sure I would read this and realize that it was that origin.